tests/test_middleware_flask.py

```
@@ -0,0 +1,390 @@
+    def test_callback_with_provider_error_redirects_to_login_not_500(self):
+        app, auth, client = _build_app()
+
+        with app.test_client() as tc:
+            resp = tc.get("/callback?error=access_denied", follow_redirects=False)
+
+        self.assertEqual(resp.status_code, 302)
+        self.assertEqual(resp.headers["Location"], "/login")
+        client.authenticate_with_code.assert_not_called()
+
+    def test_callback_with_missing_code_redirects_to_login(self):
+        app, auth, client = _build_app()
```

Ruff (0.16.1)

[warning] Unpacked variable `auth` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)